test/com/ableton/JenkinsMocksTest.groovy

```diff
@@ -13,4 +14,12 @@ class JenkinsMocksTest extends BasePipelineTest {
   void setUp() {
     super.setUp()
   }
+
+
+  @Test
+  void isUnix() throws Exception {
```

Comment:
The function starts with `is` but doesn't return a boolean...

Reply:
This conforms to JUnit naming guidelines. We are testing the `isUnix` method, and the `@Test` annotation tells us that. Previous to the introduction of that annotation (which was added in JUnit 4.0, IIRC), this would be declared as `void testIsUnix() throws Exception`, which ended up producing very verbose and redundant test code. 😉 Since we are using `@Test` it is not necessary to name this method `@Test void testIsUnix()`.

Also, JUnit test methods are required to return `void`.

Reply:
All good then!